models/wrf/readme.rst

```diff
@@ -4,34 +4,49 @@ WRF
 Overview
 --------
 
-
-DART interface module for the Weather Research and Forecasting
+The following is a description of the DART interface module for the 
+Weather Research and Forecasting model
 `(WRF) <https://www.mmm.ucar.edu/weather-research-and-forecasting-model>`__
-model. This page documents the details of the
-module compiled into DART that interfaces with the WRF data in the state vector.
+model. This page provides an overview of the module compiled into DART 
+that interfaces with the WRF data in the state vector.
 **The WRF-DART interface is compatible with WRF versions 4 and later, and is 
 no longer backwards compatible with WRFv3.9 and earlier.**  
 For more information on the interface changes required between 
-different WRF versions see the WRF tutorial link in the next section.
+different WRF versions, read through this documentation *and* the 
+WRF-DART tutorial link in the next section.  
+
+There have been several important updates to the WRF-DART interface starting
+with `DARTv11.5.0. <https://github.com/NCAR/DART/releases/tag/v11.5.0>`__ 
+Some important WRF-DART updates include:
+
+- Version 11.4.1: Detects use of the Hybrid Vertical Coordinate system
+  (terrain following at surface) and accounts for this in the forward
+  operator calculations.
+
+- Version 11.5.0: Improves compatibility with WRFv4+ versions where
+  the prognostic 3D temperature variable is THM.
+
+It is always recommended that you update your DART version to the 
+`latest release <https://github.com/NCAR/DART/releases>`__ before beginning new research.
 
-WRF+DART Tutorial
+WRF-DART Tutorial
 -----------------
 
-**There is additional overview and tutorial documentation for running a WRF/DART
-assimilation in** :doc:`./tutorial/README`
+This tutorial provides a real-world example of assimilating a wide variety of atmospheric
+observations during an extreme storm event for the United States during April 2017.
+**It is strongly recommended that you also review and perform the tutorial for 
+running a WRF-DART assimilation** `here. <https://docs.dart.ucar.edu/en/latest/models/wrf/tutorial/README.html>`__
 
-Please work through the tutorial in order to learn how to run WRF and DART.
 
-Items of Note
-~~~~~~~~~~~~~
+WRF Interface Overview
+----------------------
 
-- The ``model_mod`` reads WRF netCDF files directly to acquire the model state
-  data. The ``wrf_to_dart`` and ``dart_to_wrf`` programs are no longer
-  necessary.
-- A netCDF file named ``wrfinput_d01`` is required and must be at the same
+- The WRF ``model_mod.f90`` file reads WRF netCDF files directly to acquire the model state
+  data. Earlier versions required ``wrf_to_dart`` and ``dart_to_wrf`` programs which
+  are no longer necessary.
```

Comment:
This looks good on my end.  There are 2 changes that could lead to changes in results within the WRF model_mod: 1) the corrected conversion from vapor mixing ratio to specific humidity and 2) when a temperature observation is defined as a 'surface' obs it will interpolate on the T2 WRF variable as default, instead of interpolating on bottom level of THM/T.  Agreed  this PR description does not do a great job of describing the bug fixes-- which were found during the documentation process. 

The obs_diag.csh fix, in hindsight, is unrelated -- but is a bug that needed to be fixed while running the WRF tutorial -- which is the test case for these changes.   I was able to implement the WRF tutorial successfully.  Also included a fix with the WRF driver.csh code which likes to resubmit assim_advance.csh jobs spuriously.  Seems to be related with transition to Derecho and I previously tried to address it with sleep statements which was suboptimal and didn't always work. I turned off the resubmission capability and replaced with a warning instead.

@hkershaw-brown I think I addressed all your comments so far -- if the science changing fixes need more documentation I can provide that.  It's tough to detect physical differences in the tutorial result based on the surface temperature updates because of the brevity of the tutorial (2 assimilation cycles).  If I should reorganize the PR description or organization, let me know......